src/ladder/domain/User.java

```diff
@@ -0,0 +1,40 @@
+package ladder.domain;
+
+public class User {
+
+	private static final int MAX_USER_NAME_LENGTH = 5;
+	private static final String BLANK = " ";
+	private final String userName;
```

Comment:
```suggestion
	private static final int MAX_USER_NAME_LENGTH = 5;
	private static final String BLANK = " ";
	
	private final String userName;
```

상수와 필드를 구분해서 작성해주시면 좋을 것 같습니다.
상수 잘 사용해주셨고, `final` 키워드의 활용이 눈에 띄네요.